web/packages/plugins/simpos-ui/src/simpos/views/report.js

```diff
@@ -22,7 +22,7 @@
 
 import styled from 'styled-components'
 import { useTable } from 'react-table'
-import Plot from 'react-plotly.js'
+import Plot from '../../../../../../node_modules/react-plotly.js/react-plotly'
```

Comment:
Why the relative import here?

Reply:
not intended. Pycharm refactor magic. Fixed!